scripts/qa_crowbarsetup.sh

```diff
@@ -3161,7 +3266,9 @@ function oncontroller_testsetup()
     nova secgroup-add-rule default icmp -1 -1 0.0.0.0/0
     nova secgroup-add-rule default tcp 1 65535 0.0.0.0/0
     nova secgroup-add-rule default udp 1 65535 0.0.0.0/0
-    nova boot --poll --image $image_name --flavor $flavor --key_name testkey testvm | tee boot.out
+    fixed_net_id=`neutron net-show fixed -f value -c id`
+    nova boot --poll --image $image_name --flavor $flavor --key_name testkey \
+        --nic net-id=$fixed_net_id testvm | tee boot.out
```

Comment:
hmm, this is slightly ugly, I kind of liked that we had only one network available.. 

can we hide the manila service network from the openstack context by making it project specific?